docs/setup/install-new.md

```diff
@@ -47,23 +49,60 @@ Click on the **Deploy to Azure** icon below to begin the infrastructure deployme
 - [Public IP Address](https://docs.microsoft.com/en-us/azure/virtual-network/virtual-network-public-ip-address)
 - [Managed Identity](https://docs.microsoft.com/en-us/azure/active-directory/managed-identities-azure-resources/overview), which will be used by [AAD Pod Identity](https://github.com/Azure/aad-pod-identity/blob/master/README.md)
 
-#### Important
-Please use the `appId`, `objectId`, and `password` values from the `az` commands above and
-paste them in the corresponding ARM template fields:
-  - paste the `appId` vaule in the `Aks Service Principal App Id` template field
-  - paste the `password` value in the `Aks Service Principal Client Secret` field
-  - paste the `objectId` value in the `Aks Service Principal Object Id` field
+1. Download the ARM template and modify the template as needed.
+    ```bash
+    wget https://raw.githubusercontent.com/Azure/application-gateway-kubernetes-ingress/master/deploy/azuredeploy.json -O template.json
+    ```
 
-Note: To deploy an **RBAC** enabled cluster, set the `aksEnabledRBAC` field to `true`
+1. Create the parameter file.
```

Comment:
This is great.  It might be even greater if we moved this as step 3 after the `objectId=$(az ad sp show --id $appId --query "objectId" -o tsv)` command. This would make it clear why we are creating the bash vars and where these are used